packages/editor/src/store/actions.js

```diff
@@ -48,7 +43,7 @@ export const setupEditor =
 			// included as part of editor setup action. Assume edited content as
 			// canonical if provided, falling back to post.
 			let content;
-			if ( has( edits, [ 'content' ] ) ) {
+			if ( edits?.hasOwnProperty( 'content' ) ) {
```

Comment:
Do we intentionally want to disregard inherited properties? Also, using `.hasOwnProperty` on the object [opens this code up to prototype pollution errors](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty#using_hasownproperty_as_a_property_name).

Perhaps we could simply do `if ( 'content' in edits )`?

Reply:
Good tip, thanks. Addressed in https://github.com/WordPress/gutenberg/pull/42502/commits/9f393faefe33993505778d9b20c94b3e68b68715